src/shared-components/indicator/style.ts

```diff
@@ -3,8 +3,8 @@ import styled from '@emotion/styled';
 import { style as TYPOGRAPHY_STYLE } from '../typography';
 import { SPACER } from '../../constants';
 
-export const IndicatorContainer = styled.div`
-  background-color: ${({ theme }) => theme.COLORS.error};
+export const IndicatorContainer = styled.div<{ backgroundColor?: string }>`
```

Comment:
you could remove the `?` optional now since we'll always have a value.

If you want this to looks right in your tests. You could fix the border radius issue now in line 19 below please.
mentioned here https://app.asana.com/0/1156667858696586/1199679191263525

just set to border-radius to 50%

Reply:
I can remove `?`.

The border-radius draws from the theme, do you want me to hardcode to 50% in `/indicator/style.ts`?

```
  border-radius: ${({ theme }) => theme.BORDER_RADIUS.medium};
  ```

Or do you want me to update `SECONDARY_BORDER_RADIUS.medium` to be 50%?

Reply:
@jenniferhm We should hard-code it for the secondary theme. The secondary theme should not have any `border-radius` but there may be component-level exceptions to this rule: [see Figma link.](https://www.figma.com/file/FjOFrQaYVSdlaMIhpgR192/Ruth-UI?node-id=2315%3A6585&viewport=482%2C328%2C1.202054738998413)

Reply:
Okay will do!